PySpectra/ocean_optics.py

```diff
@@ -124,13 +168,24 @@ def read_metadata(self, filename):
             elif line.count(":") == 1:
                 elements = line.split(":")
                 self.spectra.additional_metadata[elements[0]] = elements[1]
+            elif "\t" in line:
+                line_split = line.split("\t")
+                if len(line_split) == 2 and line_split != "Wavelengths\tIntensities":
+                    try:
+                        wv_0 = float(line_split[0])
+                        dn_0 = float(line_split[1])
+                        if self.spectra.skip_header is None:
+                            self.spectra.skip_header = i
+                    except Exception:
```

Comment:
Would specifically catch `ValueError` as this is what you would get if you try and convert a non-float value to a float.